packages/ses/src/error/assert.js

```diff
@@ -196,7 +198,11 @@ freeze(unredactedDetails);
 export { unredactedDetails };
```

Comment:
Surprised to see that I exported `unredactedDetails`. Does not seem to be imported anywhere. If we cannot yet kill the (likely unused) export, could we at least deprecate it?

In any case, feel free not to address this quirk in this PR.